cmd/config.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	guid "github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// configCmd represents the config command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Links uuid with Snyk token.",
+	Long: `config registers Snyk Token 
+	with UUID on crda server and outputs UUID.`,
+	Args: validateArgs,
+	Run:  main,
+}
+
+func init() {
+	rootCmd.AddCommand(configCmd)
+
+}
+func validateArgs(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("snyk token is missing")
+	}
+	return nil
+}
+
+func requestServer(synkToken string) string {
+	ThreeScale := viper.GetString("3scaleToken")
+	url := viper.GetString("server") + "/user?user_key=" + string(ThreeScale)
+	uuid := guid.New().String()
+	payload, err := json.Marshal(map[string]string{
+		"snyk_api_token": synkToken,
+		"user_id":        uuid,
+	})
+
+	client := &http.Client{}
+	req, err := http.NewRequest(http.MethodPut, url, bytes.NewBuffer(payload))
+
+	if err != nil {
+		fmt.Println(err)
+		panic("Unable to build request")
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	res, err := client.Do(req)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Logging library should be used.

You can use the same that has been used in this module.

Reply:
Ack.

Reply:
ack.